dev-infrastructure/Makefile

```diff
@@ -213,7 +213,10 @@ svc: svc.rg
 			configurations/svc-cluster.bicepparam \
 		--parameters \
 			persist=${PERSIST}
-	@TARGET_VERSION=$(ISTIO_TARGET_VERSION) ISTIOCTL_VERSION=$(ISTIOCTL_VERSION) TAG=$(ISTIO_TAG) scripts/istio.sh
+
+	@scripts/aks-admin-access.sh $(SVC_RESOURCEGROUP) $(PRINCIPAL_ID)
+	@scripts/aks-kubeconfig.sh $(SVC_RESOURCEGROUP) $(AKS_NAME) "$(SVC_KUBECONFIG_FILE)"
+	@KUBEONFIG=$(SVC_KUBECONFIG_FILE) TARGET_VERSION=$(ISTIO_TARGET_VERSION) ISTIOCTL_VERSION=$(ISTIOCTL_VERSION) TAG=$(ISTIO_TAG) scripts/istio.sh
```

Comment:
this is here just temporary. when we switch over to running svc-pipeline.yaml, the istio script is a dedicated post-step there.